README.md

```
@@ -0,0 +1,82 @@
+### 설치
+- Java 21 설치
+- Kafka 브로커 준비 (SASL/SCRAM-SHA-256 기본 가정)
+- Naver Book API 인증 정보 준비
+
+### 컴파일 방법
+```
+./gradlew build
+```
+
+JAR 실행:
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)